### ISSUE TITLE
inconsistent convetions in numbering subordinates

### ISSUE THREAD

**Issue by ChrisMoExp** [SWTCG/SWTCG-LACKEY#19]
In some sets (e.g. JK) the S is in the rarity column, in some sets (e.g. SOLO) it is in the Number column. Feels like this should be consistent across all sets, not sure which is best.

**Comment by MatthewLarson** [comment]
I agree... I also personally think that they should get numbers. Even if the numbers didn't show up on the card images, it would be nice to have for record keeping. Would really help with moving the card ids on the Cantina.

**Comment by ChrisMoExp** [comment]
yes numbers would be good, S01, S02 etc... maybe, could even update the card images in time.

**Comment by haec0007** [comment]
I think numbers for Subordinates would be good, at least for record keeping purposes. I'd like to get @Matt-Ferus 's opinion on how they should work in the Lackey plugin, though, since I'm pretty sure Subordinate numbering has been brought up in the past.

**Comment by ChrisMoExp** [comment]
to make this more difficult / easy - there's a card called "Sith Cultist" which is both a promo and a subordinate apparently. At least he has an "S" for rarity and a "P" for number. 

in reference to a question i ask on another issue, "does the sealed draw include subs?" - if the odds of drawing are based on the rarity then how does it reference "S" in rarity... maybe it leaves it out... in which case we are constrained by that. then it follows that we can have promos in expansions with a P in rarity that would be ignored by the draw anyway and we don't need another set.

**Comment by Matt-Ferus** [comment]
Changing the rarity of all promos to P would make sense, as (P) is a separate rarity and is considered to be "ultra rare" according to the rulebook.

Regarding Subordinates, (S) is also a rarity and is considered to be "ultra common" in the rulebook.

So, P and S cards should not show up in any packs.

Also, I'm fine with S cards having numbers in Lackey or on the Cantina, but they should remain as simply S on the official card.
Numbered cards are part of the numbered set 1/180, 2/180, etc. and S and P cards should remain numberless.

Wizards had a 1,  2,   3 for each new promo card, that basically continued through each set, but then botched it by leaving gaps in the promo card numbering.

**Comment by haec0007** [comment]
I think having S and P as the rarities of subordinates and promos makes sense, and it solves issue #25. The only question is, what is the rarity of cards that are both subordinates AND promos?

**Comment by MatthewLarson** [comment]
I'm not sure if double character rarity works, but if it does, we could use SP. If it doesn't we could use U or X.

**Comment by Matt-Ferus** [comment]
As the highest rarity, promo would override the S rarity, just as promo overrides the default rare, uncommon, common, etc.

There are only a couple Subordinate Promos that have been made. But taking one as an example:
Phantom (A) with the S rarity would stay that way. 
Phantom (A) (Promo) with the P rarity would remain P.

**Comment by haec0007** [comment]
I have begun working on this in conjunction with fixing the [tap] and [Pilot] inconsistencies mentioned in #44 . I am moving the "S" to the Rarity field. In the Number field, I am sequentially numbering each Subordinate, continuing across sets. That is, the first Subordinate alphabetically in TOR is 1 and the last Subordinate alphabetically in KAE is 318.
